jck/jtrunner/config/jck11/runtime.jti

```diff
@@ -38,7 +38,7 @@ jck.env.runtime.fp.floatMinExponent=
 jck.env.runtime.fp.format=Intel, 15 bit exponent
 jck.env.runtime.fp.supportExtended=Yes
 jck.env.runtime.fp=Yes
-jck.env.runtime.ftpSupport=No
```

Comment:
Is there any document that talks about how `jck.env.runtime.ftpSupport` is used by javatest harness? Which tests would be impacted by this change? Do we need to run some tests at our end to ensure nothing is broken by this change?